### DISCUSSION TITLE
Support for Almalinux 8/9, and backup/restore scripts missing

### DISCUSSION THREAD

**Discussion by ZetaWaves** [rustdesk/rustdesk-server-pro#259] [Q&A]
Hello,

The current install script does not appear to support AlmaLinux 8 or 9. Is this going to be supported in the future now that CentOS 7 and 8 are EOL? 

Also, it seems the backup/restore scripts have been removed? What is the proper procedure to do a backup and restore now? Has this changed?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/discussions/184#discussioncomment-8013375
